sklearn/metrics/tests/test_classification.py

```diff
@@ -724,9 +725,48 @@ def test_cohen_kappa():
     )
 
 
-def test_matthews_corrcoef_nan():
-    assert matthews_corrcoef([0], [1]) == 0.0
-    assert matthews_corrcoef([0, 0], [0, 1]) == 0.0
```

Comment:
We should let this test for the moment because we did not modify anything in the MCC score.